Library/Homebrew/test/testing_env.rb

```diff
@@ -20,6 +20,25 @@
   abort "Run `bundle install` or install the mocha and minitest gems before running the tests"
 end
 
+if ENV["HOMEBREW_TESTS_COVERAGE"]
+  # Ensure all Ruby files are `require`d for an accurate coverage measurement
+  Dir[File.expand_path("../../**/*.rb", __FILE__)].each do |f|
+    next if [
+      # scripts
+      "build.rb", "postinstall.rb", "test.rb",
+      # mocked files
+      "config.rb", "global.rb",
```

Comment:
Sorry, I mean having these files in directories that indicates they are e.g. scripts/configuration/etc.